cmd/traffic/cmd/manager/service.go

```diff
@@ -173,6 +175,17 @@ func (m *Manager) GetTelepresenceAPI(ctx context.Context, e *empty.Empty) (*rpc.
 func (m *Manager) ArriveAsClient(ctx context.Context, client *rpc.ClientInfo) (*rpc.SessionInfo, error) {
 	dlog.Debug(ctx, "ArriveAsClient called")
 
+	var (
+		mode               config.Mode
+		currentClientCount = m.state.CountAllClients()
+	)
+	m.config.RLock()
+	mode = m.config.Mode
+	m.config.RUnlock()
+	if 0 < currentClientCount && mode == config.ModeSingle {
+		return nil, status.Errorf(codes.FailedPrecondition, "additional client connections require the traffic-manager to be set to team mode")
+	}
```

Comment:
this seems like it would block user #2 from connecting, which is not what we want. We should instead just nag the user